src/main/java/twilightforest/biomes/TFBiomeDecorator.java

```diff
@@ -158,13 +158,16 @@ protected void genDecorations(BiomeGenBase biome) {
         }
 
         // random features!
-        if (randomGenerator.nextInt(6) == 0) {
-            int rx = chunk_X + randomGenerator.nextInt(16) + 8;
-            int rz = chunk_Z + randomGenerator.nextInt(16) + 8;
-            int ry = currentWorld.getHeightValue(rx, rz);
-            if (ry < 75) {
-                TFGenerator rf = randomFeature(randomGenerator);
-                rf.generate(currentWorld, randomGenerator, rx, ry, rz);
+        // now with chance
```

Comment:
The "now" part of "now with chance" isn't going to be very helpful to future readers of the code, for which that comment will always have been there.

If the intent of the comment is to indicate that there's a chance of the feature not happening then you might want to encode that knowledge about the code's intent into a well named function instead.